README.md

```diff
@@ -8,6 +8,62 @@ a threshold.
 The ultimately use case for this script is to run it periodically and use it to
 trigger alerts/notifications that the Product Delivery team can take heed of.
 
+## Architectural Overview
+
+### In Plain English
```

Comment:
This title could imply that a diagram is coming next :) What do you think of *In Brief*?

Reply:
We can bikeshed and neither of us are native-tongue English speakers. I like "In Plain English" but I also don't care that much. 

The *intention* of this block is to answer, if a colleague asks, "What is this thing?"